tests/test_dataset.py

```diff
@@ -583,9 +584,10 @@ def test_only_transform_graphdataset(self):# noqa: MC0001, pylint: disable=too-m
 
     def test_only_transform_all_graphdataset(self):# noqa: MC0001, pylint: disable=too-many-locals
         # define a features_transform dict for only transformations for `all` features
+        
         hdf5_path = "tests/data/hdf5/train.hdf5"
-        features_transform = {'all': {'transform': lambda t: np.log(t+10)}}
-
+        features_transform = {'all': {'transform': lambda t: np.log(t+65)}}
+        
```

Comment:
```suggestion
        features_transform = {'all': {'transform': lambda t: np.log(abs(t)+.01)}}
        
```

I think this is slightly nicer, because
1) the 65 is kind of random (and could fail if ever we use different test data)
2) the `abs` statement makes it clearer why `t` needs to be 'pre-transformed' in the first place

Reply:
Nice